### ISSUE TITLE
Parcels stated height does not match actual height

### ISSUE THREAD

**Issue by damo1884** [cryptovoxels/issues#25]
Parcels have 2 levels of blocks underground which makes them 2 blocks shorter

See: https://www.cryptovoxels.com/parcels/2219
**Notice build height is 6m (12 blocks)**

In world: https://www.cryptovoxels.com/play?coords=W@204W,356S
**Notice total build height is 10 blocks or 5m** 

https://www.cryptovoxels.com/play?coords=W@204W,356S

**Comment by bnolan** [comment]
Yup - I realise this is a bit of a misnomer. The parcel is actually the height it says, but the bottom two rows are forced to be ground, so that we don't have holes to fall through the world. This will be fixed by making the bottom two layers of world diggable. Depends on a proper road geometry creator (currently we have a ground plane over the entire city, need to create roads as geometry).

**Comment by damo1884** [comment]
Downgrading since its a big change and its always been this way.

**Comment by damo1884** [comment]
This issue is now more an issue on islands with vertical units

6m units are 5m of build area
